lib/network.sh

```diff
@@ -118,6 +118,8 @@ else
     exit 1
 fi
 
+export DHCP_ENABLED=${DHCP_ENABLED:-"true"}
```

Comment:
It'd be good to update https://github.com/metal3-io/metal3-dev-env/blob/master/vars.md